frontend/src/fixtures/menuItemReviewFixtures.js

```diff
@@ -0,0 +1,41 @@
+const menuItemReviewFixtures = {
+    oneMenuItemReview = [
```

Comment:
Line 2 and 13, don't use equal signs. Replace with colons. 

Then run `npx eslint --fix . ` to double check

Reply:
Don't move on to new issues if previous issue doesn't pass github checks